mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -0,0 +1,479 @@
+#!/usr/bin/env python3
+
+
+import csv
+import getopt
+import os
+import subprocess
+import sys
+import math
+from prettytable import PrettyTable
+from prettytable import from_csv 
+from datetime import date
+
+# global variables.
+mlirBuildDir = './bin'
+mlirMIOpenDriver = 'mlir-miopen-driver'
+mlirROCmRunner = 'mlir-rocm-runner'
+rocprof = '/opt/rocm/bin/rocprof'
+MIOpenDriver = os.path.expanduser('~/MIOpen/build/bin/MIOpenDriver')
+benchmarkingResultFileName = 'results.stats.csv'
+configurationFileName ='../mlir/utils/jenkins/miopen-tests/resnet50-miopen-configs'
+roundDigits = 2
+
+Directions = ['-F 1', '-F 4']
```

Comment:
Style comment: these should be all caps, as in `DIRECTIONS`. Capitalized names in Python are classes and types, generally, not constants.